### ISSUE TITLE
Add https://search.rabbit-company.com

### ISSUE THREAD

**Issue by zigazajc007** [searxng/searx-instances#46]
### General

- [X] It is my instance.
- [X] I give the right to check.searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).

### Bot protection

Yes, I have installed filtron (rules.json from searx-docker).

**Comment by zigazajc007** [comment]
![Screenshot from 2022-04-17 18-22-16](https://user-images.githubusercontent.com/44822563/163723374-ecdf2b86-f185-402c-979a-a56ac8cd76b7.png)
I have just notice that additional https:// is added in the url by default.

**Comment by zigazajc007** [comment]
![Screenshot from 2022-04-17 18-24-03](https://user-images.githubusercontent.com/44822563/163723437-b1119313-e470-4b8f-9379-7be51c95e601.png)
It seems that in docker-compose.yml file https:// needs to be removed.

**Comment by tiekoetter** [comment]
I can't connect to your instance.

``ERR_CONNECTION_REFUSED``

**Comment by zigazajc007** [comment]
I'm trying to solve error above. So I'm restarting the containers.

**Comment by tiekoetter** [comment]
<img width="400" alt="Bildschirmfoto 2022-04-17 um 18 39 22" src="https://user-images.githubusercontent.com/77782671/163723997-e8764fec-f8f9-4329-a8ee-082d8acb184e.png">
Could you share your Morty config?

**Comment by zigazajc007** [comment]
Where is Morty config located? Inside Morty container?

**Comment by tiekoetter** [comment]
No I mean where you hook Morty into SearXNG.

**Comment by zigazajc007** [comment]
I have used default docker-compose.yml file.

**Comment by zigazajc007** [comment]
https://github.com/searxng/searxng-docker

**Comment by zigazajc007** [comment]
Images from Morty are working if I remove https// from the url.

**Comment by tiekoetter** [comment]
How did you configure Morty in your Webserver?

What do you want to use Morty for? If you only want an image proxy you can ignore morty and use the internal image proxy.

**Comment by tiekoetter** [comment]
> https//

This is not a valid prefix why did you add it?

**Comment by zigazajc007** [comment]
The problem is that I didn't add that. It appeared there by default. I would need to take a nother look just to make sure everything is okay.

**Comment by tiekoetter** [comment]
What webserver are you using in front of your instance? Also please share how you have added Morty there.

**Comment by zigazajc007** [comment]
https://github.com/searxng/searxng-docker/blob/master/docker-compose.yaml

I have the same docker-compose.yml file. Morty is by default included there.

**Comment by tiekoetter** [comment]
ping @unixfox

**Comment by zigazajc007** [comment]
Morty seems to be working fine. But SearXNG sends wrong URL.

```
https://search.rabbit-company.com/morty/?mortyurl=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb%2F3%2F34%2FIkesbunny.jpg%2F500px-Ikesbunny.jpg&mortyhash=b664ed5acbbaf1b0e5e2fbed9202c1ae4b7a81416909417f2306bd429f51b439
```
Morty has image located in the correct url.

But SearXNG ask for the image on wrong url:
```
https://https//search.rabbit-company.com/morty/?mortyurl=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb%2F3%2F34%2FIkesbunny.jpg%2F500px-Ikesbunny.jpg&mortyhash=b664ed5acbbaf1b0e5e2fbed9202c1ae4b7a81416909417f2306bd429f51b439
```
Like this https// should be somehow removed. I don't know if this is possible thru config files or it's hard codded inside docker container.

**Comment by zigazajc007** [comment]
This problem has been temporarly "solved" by disabling Morty and image proxy.

**Comment by unixfox** [comment]
If you found a bug in searxng-docker then please create an issue there: https://github.com/searxng/searxng-docker/issues

**Comment by zigazajc007** [comment]
So my instance is good even without Morty?

**Comment by unixfox** [comment]
I don't think we ever refused an instance that is not using morty so I guess if you configured correctly your filtron rules we will add your instance.

**Comment by tiekoetter** [comment]
@zigazajc007 Have you tried using the integrated image proxy to at least get the images proxied?

Regardless I also see no problem adding your instance. Please confirm that you are ok with it.

**Comment by zigazajc007** [comment]
Morty was working and images was proxied. It's just that SearXNG instance tried to retreive images from the wrong url. When I have tried correct url manually it worked.

Yes, you can put this instance as public.

**Comment by return42** [comment]
@zigazajc007 please update your instance.

**Comment by zigazajc007** [comment]
> @zigazajc007 please update your instance.

Updated

**Comment by unixfox** [comment]
Hello @zigazajc007,

This is a friendly reminder that your provider hetzner offer IPv6, it would be great if you could add IPv6 to your instance.

It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.

Here is the documentation here: https://docs.hetzner.com/cloud/servers/primary-ips/primary-ip-configuration/

The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).

**Comment by zigazajc007** [comment]
> Hello @zigazajc007,
> 
> This is a friendly reminder that your provider hetzner offer IPv6, it would be great if you could add IPv6 to your instance.
> 
> It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.
> 
> Here is the documentation here: https://docs.hetzner.com/cloud/servers/primary-ips/primary-ip-configuration/
> 
> The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).

I'm not sure if it's as straightforward process as it seems.

Especially as SearXNG runs inside Docker and behind Nginx Proxy Manager.

https://github.com/NginxProxyManager/nginx-proxy-manager/issues/1606

So there are multiple parts that still might not have good IPv6 support.

**Comment by return42** [comment]
@zigazajc007 FYI your instance has not been updated for a while / its recommended to update regularly.

**Comment by zigazajc007** [comment]
> @zigazajc007 FYI your instance has not been updated for a while / its recommended to update regularly.

Updated

**Comment by unixfox** [comment]
Please consider joining our Matrix room for public instance maintainers by joining our Matrix room: https://matrix.to/#/#searxng:matrix.org
then pinging @\unixfox, @\dalf and @\mrpaulblack for asking to be invited to the Matrix room.
We discuss troubles managing a public instance, sharing some advices (like how to protect against bots), announcing big changes in searxng and more.

**Comment by unixfox** [comment]
A new big change has been introduced 🎉!

Please read this documentation for how to proceed: https://gist.github.com/unixfox/4e22730769540fe5b9f1943a86439761

We are going to make mandatory this new parameter by 1st December 2023. Any instance that do not have this parameter enabled won't be allowed in the public list anymore.

We can make exceptions in case you really do not want to enable our bot limiter solution, but you will have to send us proofs: clearly stating how you detect bots, how you actively deal with them and also prove that it really works.

Feel free to ask here if you have any troubles setting up this new parameter.

**Comment by unixfox** [comment]
@zigazajc007 Important step above! The deadline has already passed, if you do not conform we will have to remove your instance by the end of the year.
